### DISCUSSION TITLE
Forgot LAN IP address of DFP-34X-2C2

### DISCUSSION THREAD

**Discussion by ccy** [Anime4000/RTL960x#101] [General]
Hi,

I have a DFP-34X-2C2 PON stick.  I forgot the LAN ip address.  Just wondering how to reset the PON stick back to factory default setting?

**Comment by Anime4000** [comment]
the stick didn't have any button or procedure to reset, only way is by doing UART Serial TTL and erase mtd3 config partition

**Reply by rndm2** [reply]
Actually, it does. 

https://github.com/Anime4000/RTL960x/assets/6842505/d8d4854c-dc27-449b-a5cc-fc3259c9f518

**Reply by kevindd992002** [reply]
> Actually, it does.
> Sb44d7aa2fd804048a93afd6a61b97cecr.mp4

Did anyone try this out already?

**Comment by filimonic** [comment]
You can scan it with nmap:

Direct connect your stick SFP and your pc lan (through VLAN PVID or through media converter)

pc LAN <--> SFP media converter <--> onu stick
pc LAN <--> Switch (PVID=? port) <--> SFP (PVID=? port) <--> onu stick

Configure your LAN to have IP from expected subnet, i.e, `192.168.222.222`
Run nmap: `"c:\Program Files (x86)\Nmap\nmap.exe"  -n -sn  192.168.0.0/16`
